src/styles.css

```diff
@@ -0,0 +1,170 @@
+body {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #6d6970;
+}
+
+header {
```

Comment:
- [ ] As the previous reviewer suggests Home page header and navigation should be similar to the given mockup. Kindly align the navbar elements to the left, please consider decreasing the padding between the navigation items so the navigation looks like the wireframe

|Your design|Wireframe|
|---|---|
|![Screenshot_20220520_180908](https://user-images.githubusercontent.com/25227022/169578823-16138a86-aae4-4ed7-a9ab-f8dd1a652f7a.png)|<img width="535" alt="169545007-0c7a4d5c-88cd-4e89-9849-b36e7f3bbb1c" src="https://user-images.githubusercontent.com/25227022/169578906-c9c3ddfa-8893-404f-a8bb-f3edac34f482.png">|